aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -1100,8 +1100,11 @@ Tensor istft(const Tensor& self, const int64_t n_fft, const optional<int64_t> ho
 
   y = y.slice(2, start, end, 1);
   window_envelop = window_envelop.slice(2, start, end, 1);
-  const auto window_envelop_lowest = window_envelop.abs().min().item().toDouble();
-  if (window_envelop_lowest < 1e-11) {
+  const auto window_envelop_lowest = window_envelop.abs().min();
+  if (at::equal(
+          window_envelop_lowest.lt(1e-11),
+          window_envelop_lowest.new_ones(
+              {}, window_envelop_lowest.options().dtype(kBool)))) {
     std::ostringstream ss;
     REPR(ss) << "window overlap add min: " << window_envelop_lowest;
     AT_ERROR(ss.str());
```

Comment:
Is the following code similar to using `TORCH_CHECK`? 
```c++
if (cond) {
   AT_ERROR(msg)
}
```

Reply:
Eg.
https://github.com/pytorch/pytorch/blob/4a1783ee4c1d812953c2c059d2bafa384f4d0e92/aten/src/ATen/native/SpectralOps.cpp#L1104-L1108

Reply:
Yeah, this is how we used to do errors before the TORCH_CHECK macros were invented